apps/extension/src/webview/index.tsx

```
@@ -0,0 +1,1709 @@
+		currentView: 'kanban',
+		selectedTaskId: undefined
+	});
+
+	const [vscode] = useState(() => {
+		return window.acquireVsCodeApi?.();
+	});
+
+	const [pendingRequests] = useState(
+		new Map<
+			string,
+			{ resolve: Function; reject: Function; timeout: NodeJS.Timeout }
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)